.travis.yml

```diff
@@ -43,3 +45,13 @@ script:
     fi
 after_success:
   - coveralls
+deploy:
+  - provider: pages:git
+    skip_cleanup: true
+    verbose: true
+    token: $GITHUB_TOKEN
+    edge: true
+    local_dir: ./docs/_build/html/
+    keep_history: true
+    on:
+	condition:  "$DEP" = "[all]" -a $TRAVIS_PYTHON_VERSION = 3.8
```

Comment:
github highlights indentation and complains about missing end of line and end of file, maybe a copy/paste issue?

Reply:
fixed. b5462067211cef93dcd68d602b6514aac917bb29